cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/cube/DefaultCube.java

```diff
@@ -130,6 +130,31 @@ SELECT count('read.ops')                                     << measure name and
     return convertToQueryResult(query, resultTable);
   }
 
+  /**
+   * Deletes the data specified by {@link CubeQuery} from all the fact tables.
+   * @param query query specifies parameters for deletion. groupByTags and resolution fields are not used.
```

Comment:
and here just leave override annotation, unless there's smth specific in this implementation